### ISSUE TITLE
Boostback is freaking out even with low errorScaling.

### ISSUE THREAD

**Issue by Matt-cloud** [smoketeer/fall#1]
Unfortunately the boostBackController is not working properly for me. It seems like it likes to follow retrograde which is also freaking out.

Here is my code

```
runoncepath("0:/fall/utilities/importFall").

importFall("landingDataModel").
importFall("boostbackController").
importFall("ascentController").

local ascent is ascentController(60000).
local landingData is landingDataModel(ship:geoposition).

set radarOffset to alt:radar.

lock steering to ascent["getSteering"]().
lock throttle to ascent["getThrottle"]().

wait until ascent["completed"]().

local boostback is boostbackController(landingData, 0.4).

lock steering to boostback["getSteering"]().
lock throttle to boostback["getThrottle"]().

wait until boostback["completed"]().
```
I've tried using the passControl function but nothing changed.  Waiting for about 10 seconds before locking the steering and throttle to boostback also didn't do much. Turning on RCS didn't do much either.

Right now the only alternative probably is to use mechjeb's landing guidance and let it do the boostback burn. After that run a kos script that contains all controllers related to guided landing except for boostback.

**Comment by smoketeer** [comment]
Excuse my late response, for some reason I didn't get any notification. I've tried your code and it works as expected. Are you sure that you have [Trajectories mod](https://ksp-kos.github.io/KOS/addons/Trajectories.html) installed?

**Comment by Matt-cloud** [comment]
Yes the trajectories mod is installed, All controllers works normally except for the boostback one, the rocket will start freaking out once i give the steering controls to the boostback controller. 

May I ask how many engines you're supposed to use with this? I've tried limiting the gimbal range on the engines but that doesn't seem to work.

**Comment by LucidDan** [comment]
Hi @Matt-cloud , similar situation here...I took the example code, not sure if it was from the example but I ended up with a similar errorscaling of 0.x, but I found that I need a errorscaling number on the boostbackController much closer to the default (5), or even more than that.
